openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py

```diff
@@ -0,0 +1,48 @@
+"""Estimates Router."""
+from openbb_core.app.model.command_context import CommandContext
+from openbb_core.app.model.obbject import OBBject
+from openbb_core.app.provider_interface import (
+    ExtraParams,
+    ProviderChoices,
+    StandardParams,
+)
+from openbb_core.app.query import Query
+from openbb_core.app.router import Router
+from pydantic import BaseModel
+
+router = Router(prefix="/estimates")
+
+# pylint: disable=unused-argument
+
+
+@router.command(model="PriceTarget")
+def price_target(
+    cc: CommandContext,
+    provider_choices: ProviderChoices,
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject[BaseModel]:
+    """Price Target. Price target data."""
+    return OBBject(results=Query(**locals()).execute())
+
+
+@router.command(model="AnalystEstimates")
+def analyst(
```

Comment:
should we be more verbose: analyst_estimates?

Reply:
This function is probably better described by, `historical_earnings_consensus`

![Screenshot 2023-11-08 at 2 03 25 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/d34a326e-74cb-413f-ae2e-754eed31d43b)

Reply:
how about `obb.equity.estimates.historical` here? then the other one can stay `obb.equity.estimates.consensus`

Reply:
i've put `obb.equity.estimates.historical` and `obb.equity.estimates.consensus`. We can re-think this in another PR